### DISCUSSION TITLE
Backup Storage Size Estimate?

### DISCUSSION THREAD

**Discussion by Snake883** [garethgeorge/backrest#544] [Q&A]
Is there a Backrest/Restic feature to estimate the backup storage size?

I would like to have an storage size estimate before uploading data to my cloud backup provider.

**Comment by garethgeorge** [comment]
Hey, unfortunately this isn't an underlying capability of restic https://forum.restic.net/t/dry-run-for-backups/1800 . If the feature is implemented there it's probably something backrest can support :)

**Reply by rammjet** [reply]
The post you link is from 2019.  In 2022, restic added a --dry-run option.  With verbose, it shows size of backups and shows what it plans to backup.

[https://restic.readthedocs.io/en/latest/040_backup.html#dry-runs](url)
